test/integration/features/experimental-feature.feature

```diff
@@ -0,0 +1,47 @@
+@experimental-feature
+Feature: Experimental Feature
+  As a user I can able to enable, run and disable experimental feature
```

Comment:
As a user I can enable, run and disable experimental feature.

Reply:
+1, will fix it